### PR TITLE
Added delete method

### PR DESCRIPTION
**Pull request by gvidal**
According to the api: https://sendy.co/api, there is a missing delete method.

### CODE REVIEW

File: src/Sendy.php [hunk 1]
```diff
@@ -17,6 +17,10 @@ class Sendy
      */
     CONST URI_UNSUBSCRIBE = 'unsubscribe';
     /**
+     * Create campaign API URI
```
Comment (jiririedl) [inline src/Sendy.php]: Please change to "Delete subscriber API URL"

File: src/Sendy.php [hunk 2]
```diff
@@ -17,6 +17,10 @@ class Sendy
...
+     */
+    CONST URI_DELETE = 'api/subscribers/delete.php';
```
Comment (jiririedl) [inline src/Sendy.php]: Please refactor const name to URI_DELETE_SUBSCRIBER
Const names fits with section in https://sendy.co/api

**Review by jiririedl — CHANGES_REQUESTED** [review]
please make these small cosmetics changes and send me PR again. Great work! Thanks for contrib!

**Comment by gvidal** [comment]
Sure thing! It is done, thank you so much, didn't realized about the typo on the comment.

**Pull request merged**